developer-docs-site/sidebars.js

```diff
@@ -113,23 +114,20 @@ const sidebars = {
     },
     {
       type: "category",
-      label: "Develop Locally",
-      link: { type: "doc", id: "nodes/local-testnet/index" },
+      label: "Implement Wallet Adapter",
```

Comment:
we are missing this category on the "Build Apps" top menu

Reply:
> we are missing this category on the "Build Apps" top menu

Thanks! Yes, that is a separate file. I will update that next to match left nav if we agree to here.